### PR TITLE
Fix TryAddOptionSet

### PR DESCRIPTION
**Pull request by LucGenetier**
Was throwing a NullRefException for some connectors having option sets

### CODE REVIEW

**Comment by LucGenetier** [comment]
✅ No public API change.

File: src/tests/Microsoft.PowerFx.TexlFunctionExporter.Shared/SwaggerFileIdentification.cs [hunk 1]
```diff
@@ -128,6 +144,13 @@ public static Dictionary<string, string> LocateSwaggerFiles(string[] folders, st
             return list2;
         }
 
+        private static int GetVersion(string str)
+        {
+            // searches '<space>[v or V]<number>' pattern
+            Match m = new Regex(@"\b[vV](?<v>[0-9]+)\\").Match(str);
+            return m.Success ? int.Parse(m.Groups["v"].Value, CultureInfo.InvariantCulture.NumberFormat) : 0;
```
Comment (MikeStall) [inline src/tests/Microsoft.PowerFx.TexlFunctionExporter.Shared/SwaggerFileIdentification.cs:151]: nit- int could throw on illegal input.  Just confirming that we want to take the exception here. 
else, Do you want TryParse and return 0 in that case?

**Review by MikeStall — DISMISSED** [review]

File: src/tests/Microsoft.PowerFx.TexlFunctionExporter.Shared/SwaggerFileIdentification.cs (hunk 1 above)
Comment (LucGenetier) [reply src/tests/Microsoft.PowerFx.TexlFunctionExporter.Shared/SwaggerFileIdentification.cs:151]: "v" group can only be a set of digits
if there could be an overflow, i'll use TryParse

**Review by MikeStall — APPROVED** [review]

**Comment by LucGenetier** [comment]
✅ No public API change.

**Pull request merged**